### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by Tyler-Thomson**
Started Django restful urls
Started models
Need to add other table that inherits from Organization, for the owner of the costumes

### CODE REVIEW

**Comment by Tyler-Thomson** [comment]
I think I need to reorganize the tables to accommodate the "events" bit on the sheet you sent us pictures of. Probably a MtM relationship between Renters and Owners, with rentals as a through table

**Comment by Tyler-Thomson** [comment]
Actually, yeah, that definitely seems like the way to go

File: apps/api/models.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+class State(models.Model):
+    name = models.Charfield(max_length=2)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+class Organization(models.Model):
+    name = models.Charfield(max_length=104)
+    address = models.Charfield(max_length=104)
+    zipcode = models.Charfield(max_length=10)
+    state = models.ForeignKey(State, related_name="clients")
```
Comment (LizhangX) [inline apps/api/models.py]: Why use clients instead of organization?

Comment (Tyler-Thomson) [reply apps/api/models.py]: They're both organizations. And to us, they're both clients. If we're planning to sell this to both sides, it doesn't make a lot of sense to unilaterally call one an organization

Comment (Tyler-Thomson) [reply apps/api/models.py]: I'm making two tables, that both inherit from Organization - Renter, and Owner

Comment (LizhangX) [reply apps/api/models.py]: Got it.

**Comment by Tyler-Thomson** [comment]
Ahhhh gotcha. Yeah, I should probably rename that @LizhangX

**Pull request merged**